### DISCUSSION TITLE
How do I remove this sidebar of the commit history?

### DISCUSSION THREAD

**Discussion by abdullahq3102** [gitkraken/vscode-gitlens#2964] [Q&A]
Hi I recently made this view pop up by accident and I dont know how to close it. Please help
![image](https://github.com/gitkraken/vscode-gitlens/assets/62462707/babcb107-6f2e-4d06-b52e-de20916f177b)

**Comment by d13** [comment]
Please click the GitLens icon to Clear the File Blame

![Screen Shot 2023-10-13 at 12 14 49 PM](https://github.com/gitkraken/vscode-gitlens/assets/293282/b87cfe0a-d49c-40a7-b10e-aa1295799b46)
